src/templates/en/2019/contributors.html

```diff
@@ -315,6 +349,15 @@ <h1 class="title title-lg">
     </div>
 
     <div class="contributor-grid">
+      <a href="https://forms.gle/Qyf3q5pKgdH1cBhq5" class="contributor-join">
+        <svg xmlns="http://www.w3.org/2000/svg" width="174" height="100" viewBox="0 0 203 117">
+          <path d="M0 0l11 89 71 4.4L65.9 117l40.7-22.4L201 89l2-79z"/>
+          <text x="50%" y="47%">
+            Join the team!
```

Comment:
One last question. Is this accessible for assistive technology users? For example, does it know to read this text to describe the anchor? If not we'll need to add some aria attributes.

Reply:
`aria-label="Join the Web Almanac team"` for the anchor or `title="Join the Web Almanac team"` for the inline SVG?  🤔

Reply:
Let's label the anchor. 👍